Tempo/Profile/SideBarViewController.swift

```diff
@@ -42,6 +42,7 @@ class SideBarViewController: UIViewController, UITableViewDelegate, UITableViewD
 		FBSDKAccessToken.setCurrentAccessToken(nil)
         let appDelegate = UIApplication.sharedApplication().delegate as! AppDelegate
         appDelegate.toggleRootVC()
+		appDelegate.feedVC.refreshNeeded = true
```

Comment:
Right now, its refreshed any time the app is opened. The problem is that when people logout and then log back in, it doesn't refresh the feed (ex. if I log out from my account and my friend logs in on his account, my feed will still show up there until my friend pulls-to-refresh).

So that extra line is called only when people logout, and it triggers the refreshFeed if the FeedViewController comes back up.